debian-packager/control

```diff
@@ -13,11 +13,13 @@ Build-Depends: cmake (>= 2.8.5),
     liblzma-dev,
     libpng12-dev | libpng-dev,
     libpulse-dev,
+    libpcap0.8-dev,
```

Comment:
Small nitpick, can you place libpcap0.8-dev below liblzma-dev.
I'd like to keep the alphabetical order still intact.